### ISSUE TITLE
Can't load add-in to Fusion

### ISSUE THREAD

**Issue by lkaratun** [macmanpb/CSV-BOM#9]
Open dialog won't let me choose the `CSV-BOM.py` file. Only folder is clickable
![image](https://user-images.githubusercontent.com/17469132/166870676-267ca8ce-5a61-4d42-989b-5a19e1e313f1.png)

Has anyone experienced this by any chance?

**Comment by petternilsen** [comment]
Yes, I see the same. Selecting the folder does not make the add-in show up in the add-in dialog.
